### PR TITLE
Add docs for NF edge release installation

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: markdown/usage/installation.md [hunk 1]
```diff
@@ -23,29 +25,63 @@ mv nextflow ~/bin/
...
+* Manually download and install Nextflow from the available [assets](https://github.com/nextflow-io/nextflow/releases) on Github. See [Nextflow installation docs](https://www.nextflow.io/docs/latest/getstarted.html#installation).
+
+### Updating
+
 We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
 
 Updating nextflow is as simple as running `nextflow self-update`
 or `conda update nextflow`, depending on how it was installed.
 
-Once installed you will probably need to configure Nextflow to run on your system. For instructions, see [_Nextflow configuration_](configuration.md).
```
Comment (maxulysse) [inline markdown/usage/installation.md:47]: Why removing this link?

Comment (drpatelh) [reply markdown/usage/installation.md:47]: I just moved that whole sentence up?

Comment (maxulysse) [reply markdown/usage/installation.md:47]: :facepalm:

**Review by maxulysse — APPROVED** [review]
LGTM

**Comment by drpatelh** [comment]
Thanks @maxulysse !

**Pull request merged**